### ISSUE TITLE
use lru_dict?

### ISSUE THREAD

**Issue by nbecker** [lonelyenvoy/python-memoization#1]
You write:
```
    :param max_items: The max items can be held in memoization cache
                      * NOT RECOMMENDED *
                      This argument, if given, can dramatically slow down the performance.
```
Would it be better to use lru-dict?

**Comment by lonelyenvoy** [comment]
Sorry for replying so late 'cause I didn't pay enough attention to this project.

Yes, it's true. Actually, **```functools.lru_cache```** provided in Python 3.2+ has a better performance than my library, so I suggest you use it instead. For references, [this](https://docs.python.org/3/library/functools.html#functools.lru_cache) is the official documentation for ```lru_cache``` at python.org.

I will fix this issue within a few days and add some features to make this lib useful.

Thanks,

**Comment by lonelyenvoy** [comment]
Resolved in v0.1.4. Now available at [PyPI](https://pypi.org/project/memoization/) or [GitHub Releases](https://github.com/lonelyenvoy/python-memoization/releases/tag/v0.1.4).
